src/praisonai/praisonai/gateway/supervisor.py

```
@@ -0,0 +1,274 @@
+                status.state = ChannelState.STOPPED
+                break
+                
+            except Exception as e:
+                # Classify the error
+                is_recoverable = self._classify_fn(e, monitor.platform)
+                is_conflict = is_conflict_error(e)
+                
+                if is_conflict:
+                    # Conflict errors are fatal - another bot instance using same token
+                    status.state = ChannelState.FAILED
+                    status.last_error = f"Conflict error (fatal): {str(e)}"
```

Ruff (0.15.15)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)